js/ui/overlay/ui.overlay.js

```diff
@@ -217,9 +221,19 @@ const Overlay = Widget.inherit({
             if(options.elementAttr && !options._ignoreElementAttrDeprecation) {
                 this._logDeprecatedOptionWarning('elementAttr', createWrapperAttrDeprecationInfo());
             }
+            if('preventScrollEvents' in options) {
+                this._logDeprecatedPreventScrollEventsInfo();
+            }
         }
     },
 
+    _logDeprecatedPreventScrollEventsInfo() {
+        this._logDeprecatedOptionWarning('preventScrollEvents', (() => ({
```

Comment:
hm, isn't it the same?
```javascript
        this._logDeprecatedOptionWarning('preventScrollEvents', {
            since: '23.1',
            message: 'If you enable this option, end-users may experience scrolling issues.'
        });
```

Reply:
I'll fix it